src/components/Common/Footer.jsx

```diff
@@ -0,0 +1,10 @@
+
+import React from 'react';
+
+export default class Footer extends React.Component {
+  render() {
+    return (
+      <footer>FOOTER LOCATION</footer>
+      );
+  }
+}
```

Comment:
Hmm.. linter should have complained about missing new line at end.  Github has picked it up though.